container/snippets/create_cluster_test.py

```diff
@@ -23,7 +23,7 @@
 import create_cluster as gke_create
 
 PROJECT_ID = os.environ["GOOGLE_CLOUD_PROJECT"]
-ZONE = "us-central1-b"
+ZONE = "us-central1"
```

Comment:
That's no longer ZONE, now I guess it should be renamed to REGION.

Reply:
Makes sense, corrected together with descriptions of the attribute.